map/map_loader/src/pointcloud_map_loader/utils.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 The Autoware Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "utils.hpp"
+
+#include <fmt/format.h>
+
+bool sphereAndBoxOverlapExists(
+  const geometry_msgs::msg::Point position, const double radius, const pcl::PointXYZ position_min,
+  const pcl::PointXYZ position_max)
+{
+  if (
+    (position_min.x - radius <= position.x && position.x <= position_max.x + radius &&
+     position_min.y <= position.y && position.y <= position_max.y && position_min.z <= position.z &&
+     position.z <= position_max.z) ||
+    (position_min.x <= position.x && position.x <= position_max.x &&
+     position_min.y - radius <= position.y && position.y <= position_max.y + radius &&
+     position_min.z <= position.z && position.z <= position_max.z) ||
+    (position_min.x <= position.x && position.x <= position_max.x && position_min.y <= position.y &&
+     position.y <= position_max.y && position_min.z - radius <= position.z &&
+     position.z <= position_max.z + radius)) {
+    return true;
+  }
+  double r2 = std::pow(radius, 2.0);
```

Comment:
```suggestion
  const double r2 = std::pow(radius, 2.0);
```

Reply:
(removed this line here https://github.com/autowarefoundation/autoware.universe/pull/1938/commits/0cc613ee578104614f53b9876a7409282375ab90)